src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -138,6 +147,18 @@ module.exports = Model.extend({
     this.fetch();
   },
 
+  getStart: function () {
```

Comment:
Are `getStart` and `getEnd` methods useful?

Seems like we're not using them anywhere, and we always return the values the user set in histogram dataview instead of the _real_ values.

Reply:
It's the same case as the bins, we return whatever the user sets 🤷‍♂️

Reply:
Yes, but you're doing that in https://github.com/CartoDB/carto.js/pull/2160/files/21e1d1e97a06083f70d83f0cf00227dbdee8a308#diff-26d0be025c5b2c0c647fb0f9b7f25e76R130 instead of here.

Here you are returning the values in the histogram data, rather than the ones that the user sets.

But my point is that I couldn't find any part of the code using these methods 🤷‍♂️